README.md

```diff
@@ -13,15 +13,20 @@ This codebase is generally intended to be network-agnostic. Programs take messag
 ## Project Dependencies
 This project relies on the `libpaillier` Rust crate using the GMP backend. GMP should be available during build-time.  
 
-##  What's Implemented
+## Rust Dependencies and Versions
+
+The preferred Rust version is cargo 1.68.2. 
```

Comment:
If we have a preferred rust version, should we instead pin it using the `rust-toolchain` file?